src/generate-md.js

```diff
@@ -0,0 +1,98 @@
+import { writeFile } from 'fs';
+
+import {
+    importJson,
+} from './utils.js';
+
+
+const registry = await importJson("../address-registry.json");
+
+const section = (title, depth = 1) => `${'#'.repeat(depth)} ${title}`;
+
+const tableHeader = `| Contract | Address |\n| --- | --- |`;
+
+const declaration = (key, value) => `| ${key} | [${value}](https://etherscan.io/address/${value}) |`;
+
+const capitalize = (s) => s.charAt(0).toUpperCase() + s.slice(1);
+
+
+registry.chains.forEach(chain => {
+
+    let content = [];
+    
+    for (const [key, value] of Object.entries(chain.contracts)) {  
+        
+        if (key == "pools") {
```

Comment:
Could replace the `if` statements with a `switch`, and throw errors if the key is not recognized. Same for the Solidity generator script.